api/types/client.go

```diff
@@ -93,4 +94,5 @@ type VanClientInterface interface {
 	SiteConfigRemove(ctx context.Context) error
 	SkupperDump(ctx context.Context, tarName string, version string, kubeConfigPath string, kubeConfigContext string) error
 	GetNamespace() string
+	GetKubeClient() kubernetes.Interface
```

Comment:
My initial instinct (not strong feeling) is that this is creating a leak in a useful abstraction. I wonder if it would be better, should we wish a simpler interface, to just add a GetVersion() method with some struct returned that has only the required information. As I say, not a strong objection, just a thought.

Reply:
Agreed as updating for mock highlights loss of abstraction. In process of updating. thnx.